servers/curated-corpus-api/src/events/glean/backend-metrics.yaml

```diff
@@ -0,0 +1,99 @@
+    type: event
+    description: >
+      Recorded when a curated corpus item is reviewed. This is converted from the Snowplow
+      "reviewed_corpus_item" event. Any array or object fields are JSON-encoded into strings.
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=000000
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=000000
+    notification_emails:
+      - your-team@example.com
+    expires: never
+    # Glean events always require a 'ping' lifetime
+    send_in_pings:
+      - events
+    extra_keys:
+      object_version:
+        type: string
+        description: "Indicates old/new version."
+      approved_corpus_item_external_id:
+        type: string
+        description: "A guid that identifies the reviewed_corpus_item’s approved_corpus_item_external_id in backend systems, sometimes referred to as an approved corpus item’s external_id."
+      rejected_corpus_item_external_id:
+        type: string
+        description: "A guid that identifies the reviewed_corpus_item’s rejected_corpus_item_external_id in backend systems, sometimes referred to as a rejected corpus item’s external_id."
+      prospect_id:
+        type: string
+        description: "Identifier for the curation prospect."
+      url:
+        type: string
+        description: "URL of the reviewed corpus item."
+      loaded_from:
+        type: string
+        description: "Backend source for the item (PROSPECT, MANUAL, etc.)."
+      corpus_review_status:
+        type: string
+        description: "Decision by the curator on the item’s validity (recommendation/corpus/rejected)."
+      rejection_reasons_json:
+        type: string
+        description: "JSON-encoded array of rejection reasons."
+      action_screen:
+        type: string
+        description: "UI screen where action took place (PROSPECTING, SCHEDULE, CORPUS) or null."
+      title:
+        type: string
+        description: "Title of the reviewed corpus item."
+      excerpt:
+        type: string
+        description: "Excerpt for the reviewed corpus item."
+      image_url:
+        type: string
+        description: "URL of the main image."
+      language:
+        type: string
+        description: "Language code of the item."
+      topic:
+        type: string
+        description: "Topic of the item."
+      is_collection:
+        type: boolean
+        description: "Whether this item is a collection."
+      is_syndicated:
+        type: boolean
+        description: "Whether the item is syndicated."
+      is_time_sensitive:
+        type: boolean
+        description: "Whether the item is time-sensitive (e.g. news)."
+      created_at:
+        type: string
+        description: "Unix timestamp (in seconds) for item creation."
+      created_by:
+        type: string
+        description: "Curator who created the item."
+      updated_at:
+        type: string
+        description: "Unix timestamp (in seconds) for last item update."

```

Comment:
[Glean events](https://mozilla.github.io/glean/book/reference/metrics/event.html) support 3 types:

> type: The type of value this extra key can hold. One of string, boolean, quantity. Defaults to string. Recorded value is converted to string for transmission. Note: If not specified only the legacy API on record is available.

However, currently `quantity` doesn't seem to be supported by the Glean parser for server-side events.

@gkatre @akkomar Is `string` or `quantity` better for timestamps? If `quantity` is preferred, I can see if it's feasible to support it.

Reply:
It's an extra key which gets converted to a string for transport anyway (see https://mozilla.github.io/glean/book/reference/metrics/event.html#events).

I think in order to support quantity we'd need to have proper type conversion in https://github.com/mozilla/glean_parser/blob/main/glean_parser/templates/javascript_server.jinja2#L227 via https://github.com/mozilla/glean_parser/blob/main/glean_parser/javascript_server.py#L63-L64 like in [Go parser](https://github.com/mozilla/glean_parser/blob/main/glean_parser/go_server.py#L62-L74). I can look into this sometime next week, or feel free take a stab if you want and have time.